package.json

```diff
@@ -25,42 +24,42 @@
   "author": "Cory House",
   "license": "MIT",
   "dependencies": {
-    "whatwg-fetch": "1.0.0"
+    "whatwg-fetch": "3.0.0"
   },
   "devDependencies": {
-    "babel-cli": "6.16.0",
-    "babel-core": "6.17.0",
-    "babel-loader": "6.2.5",
-    "babel-preset-latest": "6.16.0",
-    "babel-register": "6.16.3",
-    "chai": "3.5.0",
-    "chalk": "1.1.3",
-    "cheerio": "0.22.0",
-    "compression": "1.6.2",
-    "cross-env": "3.1.3",
-    "css-loader": "0.25.0",
-    "eslint": "3.8.1",
-    "eslint-plugin-import": "2.0.1",
-    "eslint-watch": "2.1.14",
-    "express": "4.14.0",
-    "extract-text-webpack-plugin": "1.0.1",
-    "html-webpack-plugin": "2.22.0",
-    "jsdom": "9.8.0",
-    "json-schema-faker": "0.3.6",
-    "json-server": "0.8.22",
-    "localtunnel": "1.8.1",
-    "mocha": "3.1.2",
-    "nock": "8.1.0",
-    "npm-run-all": "3.1.1",
-    "nsp": "2.6.2",
```

Comment:
Node security project (NSP) is now built into npm, so you don't need to reference it or call it manually anymore. 😀